src/content/formatQuillDelta.ts

```diff
@@ -8,10 +8,12 @@ import { QuillDelta } from "../../typings";
 
 function plaintext(data: QuillDelta): string {
   if (!data) return "";
-  return data.ops.reduce((text, op) => {
-    if (typeof op.insert !== "string") return text + " ";
-    return text + op.insert;
-  }, "");
+  return data.ops
+    .reduce((text, op) => {
+      if (typeof op.insert !== "string") return text + " ";
+      return text + op.insert.replace(/(\r\n|\n|\r)/gm, " ");
+    }, "")
+    .trim();
```

Comment:
As we discussed in person, let's not do this. If this makes trouble in the client, it should remove the line-breaks itself.